benchmarks/benchmark_interpolator.hh

```diff
@@ -0,0 +1,477 @@
+/**
+ * file  benchmark_interpolator.hh
+ *
+ * @author  Alexander Goscinski <alexander.goscinski@epfl.ch>
+ *
+ * @date   22 August 2019
+ *
+ * @brief contains the data and fixtures of the interpolator benchmarks
+ *
+ * Copyright  2019 Alexander Goscinski, COSMO (EPFL), LAMMM (EPFL)
+ *
+ * rascal is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3, or (at
+ * your option) any later version.
+ *
+ * rascal is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software; see the file LICENSE. If not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+#include <functional>
+#include <map>
+#include <iostream>
+
+#include "structure_managers/structure_manager_centers.hh"
+#include "structure_managers/adaptor_strict.hh"
+#include "structure_managers/adaptor_neighbour_list.hh"
+#include "structure_managers/make_structure_manager.hh"
+
+#include "math/interpolator.hh"
+#include "benchmarks.hh"  
+#include "representations/representation_manager_spherical_expansion.hh"
+#include "json.hpp"
+
+using namespace rascal::math;
+
+// TODO(all) naming to BFixture to prevent collision with tests ok?
```

Comment:
So we call usually the fixtures in the tests, in the benchmarks we also use fixtures for the benchmarks. To avoid naming collision like I had it for the interpolator, I would name the fixtures in the benchmarks BFixtures to prevent possible collisions or weird renaming to have a different name than in the tests.